specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2018-09-01-preview/savedquery.json

```diff
@@ -0,0 +1,536 @@
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource name. This is GUID value. The display name should be assigned within properties field."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "eTag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "This will be used to handle Optimistic Concurrency. If not present, it will always overwrite the existing resource without checking conflict."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "An azure resource object"
+    },
+    "SavedQueryListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/SavedQueryResource"
+          },
+          "description": "An array of saved queries."
+        }
+      },
+      "description": "Saved query list result."
+    },
+    "SavedQueryResource": {
+      "description": "Saved Query entity definition.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Metadata describing a saved query for an Azure resource.",
+          "$ref": "#/definitions/SavedQueryProperties"
+        }
+      }
+    },
+    "SavedQueryProperties": {
+      "description": "Properties that contain a saved query.",
+      "required": [
+        "query"
+      ],
+      "properties": {
+        "timeModified": {
+          "type": "string",
+          "description": "Date and time in UTC of the last modification that was made to this saved query definition.",
+          "readOnly": true
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of a saved query."
+        },
+        "query": {
+          "type": "string",
+          "description": "KQL query that will be saved."
+        },
+        "resultKind": {
+          "description": "Enum indicating a type of saved query.",
+          "readOnly": true,
+          "default": "basic",
+          "type": "string",
+          "enum": [
+            "basic",
+            "extended"
```

Comment:
Are we supporting extended to start with?

Reply:
We are not supporting `extended` yet. Will remove it.